tests/unit/composables/PluginManager.spec.js

```diff
@@ -243,4 +250,43 @@ describe('Test composable: PluginManager', () => {
       expect(plugin.draw).toHaveBeenCalledTimes(1);
     });
   });
+
+  describe('Test function: drawNewComponent', () => {
+    it('should addComponent, draw and render', async () => {
```

Comment:
`should call addComponent, draw and render` ?

Reply:
done